From patchwork Tue Apr 25 13:16:19 2017
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [v4,4/5] ARM: dts: rockchip: enable ARM Mali GPU on rk3288-firefly
From: Guillaume Tucker <guillaume.tucker@collabora.com>
X-Patchwork-Id: 9698147
Message-Id: <fafd6334a436b0e65d13a0c5e67263a5c6fc4ebe.1493125299.git.guillaume.tucker@collabora.com>
To: Rob Herring <robh+dt@kernel.org>, Mark Rutland <mark.rutland@arm.com>,
 =?UTF-8?q?Heiko=20St=C3=BCbner?= <heiko@sntech.de>,
 Neil Armstrong <narmstrong@baylibre.com>
Cc: devicetree@vger.kernel.org,
 Guillaume Tucker <guillaume.tucker@collabora.com>,
 Sjoerd Simons <sjoerd.simons@collabora.com>, Wookey <wookey@wookware.org>, 
 linux-kernel@vger.kernel.org, linux-rockchip@lists.infradead.org,
 John Reitan <john.reitan@arm.com>,
 Enric Balletbo i Serra <enric.balletbo@collabora.com>,
 linux-arm-kernel@lists.infradead.org
Date: Tue, 25 Apr 2017 14:16:19 +0100

Add reference to the Mali GPU device tree node on rk3288-firefly.
Tested on Firefly board.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>
---
 arch/arm/boot/dts/rk3288-firefly.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-firefly.dtsi b/arch/arm/boot/dts/rk3288-firefly.dtsi
index 10793ac18599..f520589493b4 100644
--- a/arch/arm/boot/dts/rk3288-firefly.dtsi
+++ b/arch/arm/boot/dts/rk3288-firefly.dtsi
@@ -594,3 +594,8 @@
 &wdt {
 	status = "okay";
 };
+
+&gpu {
+	mali-supply = <&vdd_gpu>;
+	status = "okay";
+};
